setup.py

```diff
@@ -40,6 +40,7 @@ def get_version():
     "werkzeug",
     "PyYAML>=5.1",
     "pytz",
+    "ecdsa<0.15",
```

Comment:
Do we need this dependency?

Reply:
Hi @bblommers, when I ran `make init` I received:
```
Installed /Users/ciaran/.pyenv/versions/3.7.8/envs/moto-3.7.8-fresh/lib/python3.7/site-packages/pyasn1-0.4.8-py3.7.egg
error: ecdsa 0.15 is installed but ecdsa<0.15 is required by {'python-jose'}
make: *** [init] Error 1
```

Hence me pinning `ecdsa` to less than 0.15.

Unsure why the Python 3.7 build is failing, it looks like `make lint` is falling over, but this ran fine locally for me (on `3.7.8`)

Reply:
The Python 2 build is failing on the `f` strings, is there a preferred method of handling this?

Reply:
String concatenation is usually done with the format-function: `"{0}".format(var)`, but old-school concatenation is acceptable as well.

Black was failing due to this issue: https://github.com/psf/black/issues/1628. Locally you might still have an old version of black, that's why it was working.
The build that is starting now should work again.. 🤞

Reply:
Awesome, I did get a ping from a friend about the black issue. Fingers crossed!

Thanks for the quick replies too @bblommers

Reply:
Looks like it's failed again on the lint

Reply:
So @bblommers I've taken a look, the new `black` version requires `regex` and `click` to be bumped. If we bump these and run `make lint`:

```
75 files would be reformatted, 583 files would be left unchanged.
``` 

I'd maybe argue that the bump in black should be a separate change, this PR could pin black to the pre 20 version?

Reply:
At a slight loss on build failures now, Python 2 looks like it's maybe because of the black formatting stripping the `u`'s? 

No idea why the recent Python 3.8 build failed https://travis-ci.org/github/spulec/moto/jobs/721473224 

Would appreciate some insights into these 😄

Reply:
The 3.8 failure was intermittant, so that's typically fixed by restarting the build.

Looks like pinning black worked though - it's all green now. 👍

Reply:
Awesome!

Thanks for the merge, what's the usual turnaround from a master merge to a PyPi installable? 

Would you like me to raise and issue to bump up black and address the fun that entails?

Reply:
Turnaround is around 10 minutes! A dev-build is automatically triggered after every commit to master. 
This is now part of moto >= 1.3.15.dev1032

Black is already aware of these bugs: https://github.com/psf/black/issues/1628

Reply:
Magic! 

Looking forward to plugging it in and trying it out! I'll probably be back for some more Step Function stuff 😇